app/src/main/java/org/xbmc/kore/service/library/SyncMusic.java

```diff
@@ -151,6 +146,8 @@ private void deleteMusicInfo(final ContentResolver contentResolver,
                                where, new String[]{String.valueOf(hostId)});
         contentResolver.delete(MediaContract.AlbumGenres.CONTENT_URI,
                                where, new String[]{String.valueOf(hostId)});
+        contentResolver.delete(MediaContract.SongArtists.CONTENT_URI,
+                               where, new String[]{String.valueOf(hostId)});
```

Comment:
errr....

This delete should have been included in the PR that introduced songsArtists...

Without it, every library sync will duplicate the values on that table. And this version is on prod... I haven't seen any complaints, but this sure looks like it will be duplicating entries.